src/schemas/integrator_acl_schema.json

```diff
@@ -0,0 +1,28 @@
+{
+    "id": "/integratorAclSchema",
+    "properties": {
+        "apiKeys": {
```

Comment:
what's the use case of having multiple keys per integrator?

Reply:
migration

Reply:
synced over Slack, this will make rolling over more straightforward